app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt

```diff
@@ -317,9 +318,30 @@ class PlayerActivity : AppCompatActivity() {
                 return true
             }
         })
+        // Handles scale/zoom gesture
+        val zoomGestureDetector = ScaleGestureDetector(this, object : ScaleGestureDetector.OnScaleGestureListener {
+            override fun onScaleBegin(detector: ScaleGestureDetector): Boolean = true
+
+            override fun onScale(detector: ScaleGestureDetector): Boolean {
+                val scaleFactor = detector.scaleFactor
+                if (abs(scaleFactor - 1f) > 0.01f) {
+                    playerView.resizeMode = if (scaleFactor > 1) AspectRatioFrameLayout.RESIZE_MODE_ZOOM else AspectRatioFrameLayout.RESIZE_MODE_FIT
+                }
+                return true
+            }
+
+            override fun onScaleEnd(detector: ScaleGestureDetector) {}
+        }).apply { isQuickScaleEnabled = false }
```

Comment:
```suggestion
        }).apply { 
            isQuickScaleEnabled = false
        }
```

Reply:
I had that before, but I didn't like the layout. Maybe I just set it on the property instead of using `apply`?